Mage.Tests/src/test/java/org/mage/test/cards/enchantments/SkullclampTest.java

```diff
@@ -25,8 +26,7 @@ public class SkullclampTest extends CardTestPlayerBase {
      * 704.5n.)
      *
      */
-    
-    // This test does not work, but the example works in the game fine.
```

Comment:
I'd like to note that I manually tested this and it doesn't seem to work in-game.